src/components/Atoms/Title/index.jsx

```diff
@@ -1,10 +1,20 @@
 import styled from "styled-components";
+import BREAKPOINT from "../../../variables/breakpoint.js";
+import COLOR from "../../../variables/color.js";
+import FONTFAMILY from "../../../variables/font_family.js";
+import TEXT from "../../../variables/texts.js";
 import React from "react";
 
 export const Title = () => {
   return <StyledText>SIMPLE TODO APP</StyledText>;
 };
 
 const StyledText = styled.div`
-  color: white;
+  color: ${COLOR.WHITE};
+  font-family: ${FONTFAMILY.ROBOTO};
+  font-size: ${TEXT.L};
```

Comment:
```suggestion
  ${TEXT.L};
```
フォントサイズに関しては変数名だけで指定できるように設定されている旨が課題に記載されていますので、ご確認お願いします！

Reply:
修正しました。